core/src/main/java/de/danoeh/antennapod/core/storage/DBReader.java

```diff
@@ -45,11 +45,6 @@ public final class DBReader {
 
     private static final String TAG = "DBReader";
 
-    /**
-     * Maximum size of the list returned by {@link #getPlaybackHistory()}.
-     */
-    public static final int PLAYBACK_HISTORY_SIZE = 50;
```

Comment:
One note: Currently the `PLAYBACK_HISTORY_SIZE` is capped at `50`, but `EPISODES_PER_PAGE` is a static constant `150` in `EpisodesListFragment`.

If we'd prefer to load playback history in smaller chunks than the episode lists, I think the way to do it might be to add an abstract getter function that the subclasses of `EpisodesListFragment` can implement, returning different constants for different fragments. For now I assumed that these were both arbitrary choices and decided to keep it simple.